NOTICE

```diff
@@ -1,6 +1,6 @@
-Copyright (c) 2017-2020 Dell Inc., or its subsidiaries. All Rights Reserved.
+Copyright (c) 2021 Pravega Authors.
```

Comment:
@derekm do we have to make a similar change to the `HEADER` file so that we reflect the copyright change onto the license header of new files?

Reply:
@fpj -- Yeah I just asked the same question in my review response. Whether HEADER change goes in this PR or not, maybe we have another PR that updates all files with the old header to add "Pravega Authors" without removing Dell copyrights.

Reply:
(Updating all files with the old headers seems preferable to updating each file as it is edited.)

Reply:
Let's not mix multiple things into a single PR. opened #5697 to properly handle the copyright section.